### ISSUE TITLE
[BUG]: lgpo_reg.set_value returns None always

### ISSUE THREAD

**Issue by stavros-k** [saltstack/salt#64126]
**Description**

I'm trying to utilize the lgpo_reg to tweak some registry values that are not present in the LGPO.
But I keep getting failure.
I did a test with the module directly and I think there might be something wrong.
As you can see in the photo (sorry couldn't copy from the VM atm) the set works, but returns None (verified with get)
So the state returns Failed every time.

**Setup**
- Minion: Windows 11 Pro in Proxmox (3006.0 - Exe)
- Master: Docker (3006.0)

**Steps to Reproduce the behavior**

On the minion execute 
```
# Set value
salt-call.exe lgpo_reg.set_value "Software\Microsoft\Windows\CurrentVersion\Explorer\Advanced" "HideFileExt" "0"
# Returns: None

# Get value
salt-call.exe lgpo_reg.set_value "Software\Microsoft\Windows\CurrentVersion\Explorer\Advanced" "HideFileExt" 
# Returns value set to 0
```

**Expected behavior**

Expected to **not** return `None` when it successfuly set value.

**Screenshots**

![image](https://user-images.githubusercontent.com/47820033/233496118-70e0a637-28eb-4192-9d16-6ffab36cfec4.png)

**Versions Report**
<details><summary>salt --versions-report</summary>
(Provided by running salt --versions-report. Please also mention any differences in master/minion versions.)

```yaml
PASTE HERE
```
</details>

**Additional context**

https://saltstackcommunity.slack.com/archives/C7K04SEJC/p1682019584935709

**Comment by stavros-k** [comment]
@twangboy Pinging you as requested!
Thanks

**Comment by twangboy** [comment]
@stavros-k Thanks! I'll try to get this in 3006.1.

**Comment by twangboy** [comment]
Closed by #64175
